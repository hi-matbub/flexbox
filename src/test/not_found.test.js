import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import NotFoundPage from '../pages/not-found_page.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<NotFoundPage />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer.create(<NotFoundPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
