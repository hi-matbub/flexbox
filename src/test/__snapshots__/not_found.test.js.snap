// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
Array [
  <h1>
    404
  </h1>,
  <p>
    Whoops, that wasnt there!
  </p>,
  <a
    href="/flexbox"
  >
    Go Home
  </a>,
]
`;
